test/e2e_test.go

```diff
@@ -63,13 +68,43 @@ func (e *e2e) TestSeccompOperator() {
 			"Re-deploy the operator",
 			e.testCaseReDeployOperator,
 		},
-	} {
+	}
+	for _, testCase := range testCases {
+		e.logf("> Running testcase: %s", testCase.description)
+		testCase.fn(nodes)
+	}
+
+	// Clean up cluster-wide deployment to prepare for namespace deployment
+	e.cleanupOperator(manifest, defaultProfiles)
+
+	e.logf("testing namespace operator")
+
+	// Use namespace manifests for redeploy test
+	testCases[4].fn = e.testCaseReDeployNamespaceOperator
+
+	// Deploy the namespace operator
+	e.kubectl("create", "namespace", testNamespace)
+	e.run(
+		"sed", "-i", fmt.Sprintf("s/NS_REPLACE/%s/", testNamespace),
+		namespaceManifest,
+	)
+	e.run(
+		"sed", "-i", fmt.Sprintf("s/NS_REPLACE/%s/", testNamespace),
+		namespaceDefaultProfiles,
+	)
+	defer e.run("git", "checkout", namespaceManifest)
+	defer e.run("git", "checkout", namespaceDefaultProfiles)
+	// All following operations such as create pod will be in the test namespace
+	e.kubectl("config", "set-context", "--current", "--namespace", testNamespace)
```

Comment:
Potential future optimization: using unstructured client for these operations instead of kubectl directly